docker/Dockerfile

```diff
@@ -11,9 +11,7 @@ LABEL name="LSST Header Service" \
 # Pre-reqs for HeaderService
 ARG MINICONDA_PATH=/opt/lsst/software/stack/miniconda
 RUN source ${MINICONDA_PATH}/bin/activate \
-    && mamba install -y fitsio pyyaml \
-    # Need to downgrade to setuptools=59.8.0
-    && mamba install -y setuptools=59.8.0
+    && conda install -y fitsio pyyaml
```

Comment:
I would still suggest changing `conda` to `mamba` here. Cuts solving time by at least a third.

Reply:
Changed to mamba